app/pages/sections/catalogue-solutions/catalogue-solutions/controller.js

```diff
@@ -17,7 +17,7 @@ export const getCatalogueSolutionsPageContext = async ({ orderId, accessToken })
   return getContext({
     orderId,
     orderDescription: orderDescriptionData ? orderDescriptionData.description : '',
-    orderItems: addedSolutionsData.orderItems,
+    orderItems: addedSolutionsData.catalogueSolutions,
```

Comment:
just wondering why this was changed?

Reply:
because the api doesn't currently support orderItems for catalogue solutions, it still uses the old route.

Reply:
It'll need changing once the api is changed

Reply:
ah ok. good spot on that one 👍